src/main/io/asyncfatfs/asyncfatfs.c

```diff
@@ -2294,7 +2294,7 @@ static afatfsOperationStatus_e afatfs_extendSubdirectoryContinue(afatfsFile_t *d
             }
 
             // Seek back to the beginning of the cluster
-            afatfs_assert(afatfs_fseekAtomic(directory, -AFATFS_SECTOR_SIZE * (afatfs.sectorsPerCluster - 1)));
+            afatfs_assert(afatfs_fseekAtomic(directory, -(AFATFS_SECTOR_SIZE * (afatfs.sectorsPerCluster - 1))));
```

Comment:
We should probably cast `afatfs.sectorsPerCluster` to signed int. Because it's uint32 that could overflow, but since it's used like this we are already assuming that it won't.

Reply:
But it won't really change the result so I guess it's fine if just for removing warning ;)

Reply:
Updated